pkg/abaputils/abaputils_test.go

```diff
@@ -15,6 +15,7 @@ import (
 )
 
 func TestCloudFoundryGetAbapCommunicationInfo(t *testing.T) {
+	t.Parallel()
 	t.Run("CF GetAbapCommunicationArrangementInfo - Error - parameters missing", func(t *testing.T) {
```

Comment:
Do we need an additional `t.Parallel()` inside each `t.Run`?

@marcusholl @stippi2

Reply:
Yes. See https://github.com/SAP/jenkins-library/blob/master/DEVELOPMENT.md#test-parallelization